spec/preprocessorSpec.js

```diff
@@ -0,0 +1,33 @@
+    smallString: "<h1>Dit is een</h1> <h2>standaard</h2>- TEKST <ul><li>test1</li><li>test2</li><li>test3</li><li>test4</li></ul>met VEEL caps, spaties, <h6>tekens</h6> en andere overbodige meuk!?'...; <img src='http://linknaarplaatje' alt='mooiplaatje' />Het aantal Woorden<br><br> is negentien"
+};
+
+output = {
+    cleanText: "<h1>dit is een</h1> <h2>standaard</h2> tekst <ul><li>test1</li><li>test2</li><li>test3</li><li>test4</li></ul>met veel caps spaties <h6>tekens</h6> en andere overbodige meuk. <img src= http //linknaarplaatje alt= mooiplaatje />het aantal woorden<br><br> is negentien.",
+    cleanTextSomeTags: "<h1>dit is een</h1> <h2>standaard</h2> tekst <li>test1</li><li>test2</li><li>test3</li><li>test4</li> met veel caps spaties <h6>tekens</h6> en andere overbodige meuk. het aantal woorden is negentien.",
+    cleanTextNoTags: "dit is een standaard tekst test1 test2 test3 test4 met veel caps spaties tekens en andere overbodige meuk. het aantal woorden is negentien."
+};
+
+describe("preprocessor flattext output", function(){
```

Comment:
Since you are only testing one function for one testcase, you only need one describe block. I would structure the testfile as follows:

```JS
describe("PreProcessor with an HTML string", function(){
  textString = "...";
  expectedOutput = {};
  preproc = new PreProcessor(textString);

  it("returns processed clean text", function(){
    expect(preproc._store.cleanText).toBe(expected_output.cleanText);
  });

  it("returns processed notags text", function(){
    expect(preproc._store.cleanTextNoTags).toBe(expectedOutput.cleanTextNoTags);
  });

  it("returns processed sometags text", function(){
    expect(preproc._store.cleanTextSomeTags).toBe(expectedOutput.cleanTextSomeTags);
  });
});
```

Gives you a cleaner, easier to read interface without duplication.